api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/ContentModelSetup.java

```diff
@@ -191,6 +213,144 @@ private void renamePortal(Resource portal, String namespace, Model model) {
         }
     }
 
+    /*
+     * Check if the firsttime files have changed since the firsttime startup,
+     * if so, then apply the changes but not overwrite the whole user model
+     */
+    private void applyFirstTimeChanges(ServletContext ctx) {
+        log.info("Start with check: Reload firsttime files on start-up if changed");
+        boolean updatedFiles = false;
+
+        // get configuration models from the firsttime start up (backup state)
+        ContextModelAccess models = ModelAccess.on(ctx);
+        OntModel applicationMetadataModel = models.getOntModel(APPLICATION_METADATA_FIRSTTIME);
+        OntModel baseABoxModel = models.getOntModel(ABOX_ASSERTIONS_FIRSTTIME);
+        OntModel baseTBoxModel = models.getOntModel(TBOX_ASSERTIONS_FIRSTTIME);
+
+
+        // check if ApplicationMetadataModel is the same in file and configuration models
+        log.debug("compare firsttime files with configuration models (backup from first start) for Application Metadata Model");
+        OntModel testApplicationMetadataModel = VitroModelFactory.createOntologyModel();
+        RDFFilesLoader.loadFirstTimeFiles(ctx, "applicationMetadata", testApplicationMetadataModel, true);
+        setPortalUriOnFirstTime(testApplicationMetadataModel, ctx);
+    
+        if ( applicationMetadataModel.isIsomorphicWith(testApplicationMetadataModel) ) {
+            log.debug("They are the same, do nothing");
+        } else {
+            log.debug("they differ, compare values in configuration models with user's triplestore");     
+            OntModel userTriplestoreApplicationMetadataModel = models.getOntModel(APPLICATION_METADATA);
+
+            // double check the statements (blank notes, etc.) and apply the changes
+            updatedFiles = applyChanges(applicationMetadataModel, testApplicationMetadataModel, userTriplestoreApplicationMetadataModel, "applicationMetadata");
+            if (updatedFiles) log.info("The applicationMetadata model was updated.");
+        }
+
+        // check if abox model is the same in file and configuration models
+        log.debug("compare firsttime files with configuration models (backup from first start) for Base ABox Model");
+        OntModel testBaseABoxModel = VitroModelFactory.createOntologyModel();
+        RDFFilesLoader.loadFirstTimeFiles(ctx, "abox", testBaseABoxModel, true);
+
+        if ( baseABoxModel.isIsomorphicWith(testBaseABoxModel) ) {
+            log.debug("They are the same, so do nothing");
+        } else {
+            log.debug("they differ, compare values in configuration models with user's triplestore");
```

Comment:
Could you please change the logging message as follows:
```
log.debug("They differ:" + modelPath + ", compare values in configuration models with user's triplestore");
```

Reply:
i changed it